src/harness/testcases/testdata/sas_impl_record_0.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "sas_impl/0/0",
+  "name": "Sas Implementation Record",
+  "administratorId": "sas_admin/sas/0",
```

Comment:
Thank you for forwarding the email! It seems that it has been updated in v1.5.3 Spec. Should we follow the latest version v1.5.3 while creating the test cases?